src/Resource.cpp

```diff
@@ -585,6 +585,7 @@ namespace D3D12TranslationLayer
                         IID_PPV_ARGS(&m_Identity->m_spUnderlyingResource));
                 }
                 const UINT D3D11_BIND_CAPTURE = 0x800;
+                UNREFERENCED_PARAMETER(D3D11_BIND_CAPTURE);

```

Comment:
Since this is a C++17 codebase, I'd prefer to just mark `D3D11_BIND_CAPTURE` as `[[maybe_unused]]` which will suppress the warning.

Reply:
Didn't know about [[maybe_unused]].  Will switch that now.